packages/react/popper/src/Popper.tsx

```diff
@@ -135,7 +135,7 @@ const PopperContent = React.forwardRef<PopperContentElement, PopperContentProps>
       alignOffset = 0,
       arrowPadding = 0,
       avoidCollisions = true,
-      collisionBoundary = [],
+      collisionBoundary,
```

Comment:
I guess this might be a potentially breaking change, because there's a semantical difference between _"Keep boundary checks up to the component"_, when no `collisionBoundary` prop is passed, and _"I deliberately want to have no boundaries"_, when an empty array is provided.

I'm not sure whether there are legitimate use cases for passing an empty boundary list to the component, but I see [there were at least some discussions in floating-ui repo when passing an empty boundary list was intentional](https://github.com/floating-ui/floating-ui/issues/1110#issuecomment-632432704). Because I don't have much data on how people use Radix, I think it's safer to only change the default behavior, when no definite value for `collisionBoundary` is passed.